spec/models/article_spec.rb

```diff
@@ -469,17 +469,17 @@ def build_and_validate_article(*args)
     it "deindexes unpublished article" do
       expect do
         article.update(body_markdown: "---\ntitle: Title\npublished: false\ndescription:\ntags: one\n---\n\n")
-      end.to have_enqueued_job(Search::RemoveFromIndexJob).with(described_class.algolia_index_name, article.id).
-        and have_enqueued_job(Search::RemoveFromIndexJob).with("searchables_#{Rails.env}", article.index_id).
-        and have_enqueued_job(Search::RemoveFromIndexJob).with("ordered_articles_#{Rails.env}", article.index_id)
+      end.to have_enqueued_job(Search::RemoveFromIndexWorker).with(described_class.algolia_index_name, article.id).
```

Comment:
does Sidekiq have these helpers? AFAIK it is

```ruby
expect { }.to change(Worker.jobs, :size).by(1)
```

reference: https://github.com/mperham/sidekiq/wiki/Testing#testing-worker-queueing-fake

Reply:
We recently added a [bunch of helpers](https://github.com/thepracticaldev/dev.to/blob/master/spec/support/sidekiq_test_helpers.rb), I'm going to update #5305 mentioning them

Reply:
see https://github.com/thepracticaldev/dev.to/issues/5305#issuecomment-570264567